Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers_description.md

```diff
@@ -26,4 +19,24 @@ To use a self-configured Azure application, you need to add a new Azure App Regi
 *Make sure to provide the following permissions for the app to work with Azure Risky Users:*
  - ***IdentityRiskyUser.Read.All*** - https://docs.microsoft.com/en-us/graph/api/riskyuser-list?view=graph-rest-1.0
  - ***IdentityRiskEvent.Read.All*** - https://docs.microsoft.com/en-us/graph/api/riskdetection-get?view=graph-rest-1.0
- 
```

Comment:
The README lists these permissions
- *IdentityRiskEvent.Read.All*
- *IdentityRiskEvent.ReadWrite.All*
- *IdentityRiskyUser.Read.All*
- *IdentityRiskyUser.ReadWrite.All*
- *User.Read*

But this file only lists these:
 - ***IdentityRiskyUser.Read.All*** - https://docs.microsoft.com/en-us/graph/api/riskyuser-list?view=graph-rest-1.0
 - ***IdentityRiskEvent.Read.All*** - https://docs.microsoft.com/en-us/graph/api/riskdetection-get?view=graph-rest-1.0

Reply:
Added the missing permissions to the description file. Thanks.